CHANGELOG.md

```diff
@@ -0,0 +1,63 @@
+# Open Trip Planner - React Redux Changelog
+
+## [3.0.0](https://github.com/opentripplanner/otp-react-redux/releases/tag/v3.0.0) (2021-04-13)
+
+- Add mobile batch results screen
+- User Account Settings
+- Add more buttons to favorites management
+- Re-organize user settings
+- Add field trip module
+
+## [3.1.0](https://github.com/opentripplanner/otp-react-redux/releases/tag/v3.1.0) (2021-07-07)
+
+- Add reverse directions button
+- Add mailables submodule to calltaker module
+- Improve map performance
+
+## [3.2.0](https://github.com/opentripplanner/otp-react-redux/releases/tag/v3.2.0) (2021-07-15)
+
+- Add GTFS-RT vehicle overlay
+- Add user language settings
+- Support overriding strings
+
+## [3.3.0](https://github.com/opentripplanner/otp-react-redux/releases/tag/v3.3.0) (2021-07-22)
+
+- Field trip module
+- Add save button
+- Add delete button
+- Allow configuration of capacity
+
+## [3.4.0](https://github.com/opentripplanner/otp-react-redux/releases/tag/v3.4.0) (2021-08-18)
+
+- Add automatic a11y compliance tests
+- Adjust colors to conform with [Web Content Accessibility Guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/).
+
+## [3.5.0](https://github.com/opentripplanner/otp-react-redux/releases/tag/v3.5.0) (2021-10-05)
+
+- Add Traveler Tools Menu
+- Add Route Details Viewer
+- Move route viewer to title bar
+- Add support for displaying realtime vehicle positions
+- Support filtering routes by agency or mode
+
+## [3.6.0](https://github.com/opentripplanner/otp-react-redux/releases/tag/v3.6.0) (2021-10-21)
+
+- Further replace colors with accessible ones
+- Add nearby amenities panel to stop viewer
+- Add support for Local Places Index
+
+## [3.7.0](https://github.com/opentripplanner/otp-react-redux/releases/tag/v3.7.0) (2021-11-12)
+
+- Fix reported crashes
+- Zoom to stop when opening stop viewer
+- Display search results based on geolocation
+- New stop markers on map at high zoom levels
+- Add accessibility labels to legs and itineraries when accessible routing is enabled
+- Support showing multiple fare prices
+
+## [3.8.0](https://github.com/opentripplanner/otp-react-redux/releases/tag/v3.8.0) (2021-12-16)
+
+- Fix crashes reported to us
+- Improve screen reader experience
+- Add user setting for planning accessible trips by default
+- Support internationalization of most component
```

Comment:
'component' -> 'components'

Reply:
Good catch! Changed in https://github.com/opentripplanner/otp-react-redux/pull/537/commits/5e363e393de0502c385885d4aefd95d2d61cc484